apps/protocol-frontend/src/components/DaoTextareaForm.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+import { Button, Flex, Icon, Text } from '@chakra-ui/react';
+import { AiFillExclamationCircle, AiFillCheckCircle } from 'react-icons/ai';
+import { ControlledTextarea } from '@govrn/protocol-ui';
+import { splitEntriesByComma } from '../utils/arrays';
+import { isEthAddress } from '../utils/validations';
+import { isArray } from 'lodash';
+
+const DaoTextareaForm = () => {
+  const [daoMemberAddresses, setDaoMemberAddresses] = useState('');
+  const [validationError, setValidationError] = useState<string | null>(null);
+
+  const handleDaoTextAreaChange = (
+    e: React.ChangeEvent<HTMLTextAreaElement>,
+  ) => {
+    if (validationError !== null) {
+      setValidationError(null);
+    }
+    setDaoMemberAddresses(e.target.value);
+    validateDaoMemberAddresses();
+  };
+
+  const validateDaoMemberAddresses = () => {
```

Comment:
This fails if there is a single address and no comma. It says invalid address. maybe the message should be different.

The same error occurs if the last address does not end in a comma

Reply:
Ah yeah. Going to rework the validation - I'm not happy with it and it's finicky as it is. I have a version I'm trying that does use RHF and a custom yup validator instead since this ended up being likely  more work to not use it

Reply:
Updated to use RHF and the validation is smoother. I was going to do that originally but thought we'd run into issues with the validation. The custom yup `.test` is working to validate each address in the list